src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_ev_driver_posix.c

```diff
@@ -38,7 +38,7 @@ typedef struct fd_node {
   /** the owner of this fd node */
   grpc_ares_ev_driver *ev_driver;
   /** the grpc_fd owned by this fd node */
-  grpc_fd *grpc_fd;
+  grpc_fd *fd;
```

Comment:
The original name makes it clearer that this is a `grpc_fd` object, not a raw file descriptor.  Please stick with the original name here.

Reply:
The issue here is that we cannot have the same type and variable name in C++.